projects/components/src/menu-dropdown/menu-dropdown.component.ts

```diff
@@ -8,7 +8,7 @@ import { MenuItemComponent } from './menu-item/menu-item.component';
   styleUrls: ['./menu-dropdown.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
-    <div class="menu-dropdown">
+    <ht-event-blocker event="click" class="menu-container" [enabled]="this.disabled">
```

Comment:
same class for parent and child?

also, why are there multiple PRs modiying the same thing?

Reply:
yeah fixing the css. As to multiple PRs, just needed to change its base. Create separate PRs since they were touching different components